lib/std/os/linux.zig

```diff
@@ -1522,6 +1522,12 @@ pub fn signalfd(fd: fd_t, mask: *const sigset_t, flags: u32) usize {
     return syscall4(.signalfd4, @bitCast(usize, @as(isize, fd)), @ptrToInt(mask), NSIG / 8, flags);
 }
 
+pub fn clone_file(fd_in: fd_t, fd_out: fd_t) usize {
+    assert(fd_in >= 0);
+    assert(fd_out >= 0);
+    return ioctl(fd_out, T.FICLONE, @bitCast(usize, @as(isize, fd_in)));
+}
+
```

Comment:
This file is for raw syscalls and should not have any higher level abstractions in it. Check `std.os.ioctl_SIOCGIFINDEX` for an example of how you could create such a "clone file" abstraction. This will additionally ensure that the call goes through libc `ioctl` when the application links libc.